src/MPI.jl

```diff
@@ -32,32 +35,22 @@ function __init__()
         end
     end
 
-    global const mpitype_dict = Dict{DataType, Cint}(
-        # Older versions of OpenMPI (such as those used by default in
```

Comment:
Whoops, I missed this note.   I may not have time to do this today, and I'll be away for a few days on vacation soon.  Could you push a patch to this branch restoring the comment?